### DISCUSSION TITLE
Copilot student

### DISCUSSION THREAD

**Discussion by RobbeH0Gent** [community/community#28187] [Copilot Conversations] [answered]
I activated my copilot (without knowing there was a date that the free trail ended), the issue is: I 've got my exam of web development where I use copilot for only the 7th of september but my free trial stops at 22 august. Is there anyway I can use the free copilot till the 8th?

**Comment by devtayyab** [comment]
You can login with any another git account , and one option is change your GitHub username and then login again

**Comment by RobbeH0Gent** [comment]
doesnt work for me, still the same
due date



________________________________
Van: Muhammad Tayyab ***@***.***>
Verzonden: woensdag 3 augustus 2022 16:36
Aan: community/community ***@***.***>
CC: Robbe Schoukens ***@***.***>; Author ***@***.***>
Onderwerp: Re: [community/community] Copilot student (Discussion #28187)


You can login with any another git account , and one option is change your GitHub username and then login again

**Comment by davecheney** [comment, marked as answer]
@RobbeH0Gent if you have an active student coupon you can use that to sign up for copilot for the duration of your coupon.

**Reply by gvicenteprieto** [reply]
Cómo se carga el comprobante de estudiante activo? desde ya, muchas gracias

**Reply by gvicenteprieto** [reply]
How is the active student voucher uploaded? Thanks a lot

**Reply by davecheney** [reply]
I have not used the system myself so I cannot comment but I would imagine the notification you received with your student coupon will contain instructions for how to apply the coupon.
